src/V3Number.h

```diff
@@ -333,11 +673,10 @@ class V3Number final {
     bool isEqAllOnes(int optwidth = 0) const;
     bool isCaseEq(const V3Number& rhs) const;  // operator==
     bool isLtXZ(const V3Number& rhs) const;  // operator< with XZ compared
-    void isSigned(bool ssigned) { m_signed = ssigned; }
     bool isAnyX() const;
     bool isAnyXZ() const;
     bool isAnyZ() const;
-    bool isMsbXZ() const { return bitIsXZ(m_width); }
+    bool isMsbXZ() const { return bitIsXZ(m_data.width() - 1); }
```

Comment:
The - 1 appears to be fixing a bug, so please make a pre-pull request with that -1 so we can confirm nothing breaks with it and separate it from any potential bugs in your new implementation thanks.

Reply:
Fixed in https://github.com/verilator/verilator/pull/3530